modules/social_features/social_event/social_event.install

```diff
@@ -153,6 +153,20 @@ function social_event_update_121002(): string {
   return $updateHelper->logger()->output();
 }
 
+/**
+ * Ensure the sidebar block "Upcoming events" will be displayed on all groups.
+ */
+function social_event_update_121003(): string {
```

Comment:
Since we plan to release it in version 12.2.x it's better to rename the hook update number to the `_122001`